wsdl/ver10/schema/onvif.xsd

```diff
@@ -7693,6 +7693,16 @@ and sample rate. </xs:documentation>
 					</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="CertificateId" type="tas:CertificateID" minOccurs="0" maxOccurs="unbounded">
```

Comment:
One minor thing: As we now have this field for the certificate ID, should we change the type of KeyID back to `tas:hexBinary`, as we won't be using it for anything other than the key ID stored in the `pssh` box?

Reply:
done